pkg/skaffold/sync/sync.go

```diff
@@ -247,7 +247,7 @@ func Perform(ctx context.Context, image string, files map[string]string, cmdFn f
 
 				cmds := cmdFn(ctx, p, c, files)
 				for _, cmd := range cmds {
-					if err := util.RunCmd(cmd); err != nil {
+					if _,err := util.RunCmdOut(cmd); err != nil {
```

Comment:
you are not doing anything with the output bytes? Is this required?

Reply:
@tejal29  should we print the stdout as well?

Reply:
yes, if you want to show the output bytes.

Reply:
`stdout` should be uninteresting in this case here. It's usually the output of `rm` or `tar xf` and would only clutter the log IMHO.

Reply:
agree, the only reason I can see needing it is for debugging, but even then I think any failures should surface through `stderr`.